### DISCUSSION TITLE
New Update - Am I missing something??

### DISCUSSION THREAD

**Discussion by MajorKryme** [SavedInstances/SavedInstances#484] [Q&A]
Hello,

First, I love this addon.  Second, I just updated to 9.0.6 and now when I mouseover the minimap icon, I'm seeing no data...no currencies, etc.  I used to see all my server alts and soul ash, lockouts, etc.  Today, I see none of that unless I use the "alt" key to bring up the full data window.

Am I missing a setting or something?  I'm sure it's trivial, so I'm hoping you can help me out.

Thanks!

**Comment by LiangYuxuan** [comment]
Would you check out your settings and find if there're something you want but not checked?

**Reply by MajorKryme** [reply]
I just saw this message, but about halfway through raid, I guess the addon updated itself or whatever, because everything I was seeing prior to the update, was now populated.  It appears to be all good and I made zero option changes...maybe I just needed to give it a bit of time.  Thank you.
